modules/_labs/tabs/react/lib/cursor/useCursorModel.tsx

```diff
@@ -0,0 +1,149 @@
+import React from 'react';
+import {
+  createEventMap,
+  Model,
+  ToModelConfig,
+  useEventMap,
+  assert,
+} from '@workday/canvas-kit-react-common';
+
+import {
+  ListState,
+  ListEvents,
+  BaseListModelConfig,
+  listEventMap,
+  useListModel,
+  Item,
+} from '../list/useListModel';
+
+export type Orientation = 'horizontal' | 'vertical';
+
+export type CursorState = ListState & {
+  /** Orientation of the list. This property determines what "next" and "previous" mean. This should
+   * be set to match the layout of UI components */
+  orientation: Orientation;
+  /** The id of the list item the cursor is pointing to */
+  cursorId: string;
+};
+
+export type CursorEvents = ListEvents & {
+  /** Directly sets the cursor to the list item by its identifier. Bypasses any checks  */
+  setCursorId(data: {id: string}): void;
```

Comment:
We talked about this being `goTo`, but I don't think we landed on anything. Your call.

Reply:
Yes. I think `goTo` would work. It is hard to come up with a name that isn't too verbose, yet works in multiple contexts. This event will set the cursor to a specific point in the list, which is used by the Tabs model. In this context, `goTo` will set the focus cursor while `activateTab` will set the active tab.

Should `activateTab` be changed to `activate` and assume the `Tab` context is implied? It can be difficult with model composition to make too many assumptions about the context in which the events will be called.

Should we always or never explicitly specify model context?

Explicit model context:
- `registerListItem` - from `ListModel`
- `goToCursor` - from `CursorModel`
- `activateTab` - from `TabsModel`

Implicit model context:
- `registerItem` - from `ListModel`
- `goTo` - from `CursorModel`
- `activate` - from `TabsModel`